app/views/facility_accounts/account_fields/_nufs_account.html.haml

```diff
@@ -1 +1,6 @@
+- is_new ||= false
```

Comment:
`||=` with booleans always feel dangerous. In partials I like to use `local_assigns[:is_new]` that'll be nil if it's not set, otherwise it'll be the value you passed in.